components/layout_2020/lib.rs

```diff
@@ -3,6 +3,7 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
 
 #![deny(unsafe_code)]
+#![allow(clippy::needless_lifetimes)]
```

Comment:
Can you open a bug to remove these? Typically we have been removing lifetimes when Rust allows us to and IMO it makes the code easier to read by removing visual noise from definitions.

Reply:
https://github.com/servo/servo/issues/34804